### PR TITLE
FRJpvOMb: Replace Carthage with CocoaPods

### PR DESCRIPTION
**Pull request by pryaniq**
- add bundler
  - remove Cartfile
  - add and configure CocoaPods and dependencies
  - add build scripts
  - remove R.generated

### CODE REVIEW

**Comment by pryaniq** [comment]
[Replace Carthage with CocoaPods](https://trello.com/c/FRJpvOMb/18-replace-carthage-with-cocoapods)

**Comment by pryaniq** [comment]
[Replace Carthage with CocoaPods](https://trello.com/c/FRJpvOMb/18-replace-carthage-with-cocoapods)

**Comment by pryaniq** [comment]
[Replace Carthage with CocoaPods](https://trello.com/c/FRJpvOMb/18-replace-carthage-with-cocoapods)

**Comment by pryaniq** [comment]
[Replace Carthage with CocoaPods](https://trello.com/c/FRJpvOMb/18-replace-carthage-with-cocoapods)

File: build_scripts/swiftlint.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+if [ "${CONFIGURATION}" = "Debug" ]; then
+  echo "SwiftLint started"
+  Pods/SwiftLint/swiftlint --quiet --path "${1:-.}"
```
Comment (DaianAiziatov) [inline build_scripts/swiftlint.sh:3]: 👍
we can also add swiftformat linting on UnitTests run going forward

**Review by DaianAiziatov — APPROVED** [review]

File: build_scripts/swiftlint.sh (hunk 1 above)
Comment (pryaniq) [reply build_scripts/swiftlint.sh:3]: Agree. Any kind of linters/validators will pay off.

**Pull request merged**